src/profile-logic/import/instruments/index.js

```diff
@@ -11,6 +11,38 @@ import type {
   StackTable,
 } from '../../../types/profile';
 
+type Sample = {
```

Comment:
Generally for types we prefer them to be exact and read-only:

https://flow.org/en/docs/types/objects/#toc-exact-object-types

Adding `|` to an object makes it exact:

```
type Sample = {|
  ...
|};
```

Adding `+` makes property read-only, which protects us against unexpected mutation. (At least shallowly read-only)

`   +timestamp: number, `

Reply:
Got it. I will make the changes accordingly.